app/registeredServices.json

```diff
@@ -108,5 +108,25 @@
     "redirectLinks": {
       "jurordigital_citizen": "https://reply-jury-summons.service.gov.uk/steps/confirm-information"
     }
+  },
+  {
+    "serviceId": "prl_ca",
+    "actors": [
+      "APPLICANT",
+      "RESPONDENT"
+    ],
+    "redirectLinks": {
+      "prl_ca": "https://reply-jury-summons.service.gov.uk/steps/confirm-information"
+    }
+  },
+  {
+    "serviceId": "prl_da",
+    "actors": [
+      "APPLICANT",
+      "RESPONDENT"
+    ],
+    "redirectLinks": {
+      "prl_ca": "https://reply-jury-summons.service.gov.uk/steps/confirm-information"
```

Comment:
This should be prl_da assuming they'll have different return urls. If not then i think we can just have 1 in prl_ca and leave prl_da blank. Then the shutter page will only render the prl_ca link.

Reply:
Thanks Bradley , they have only one redirect link at this point as they both coming from same application. I'll leave this as blank. If they have different URL , then will add a new one.